packages/fondue/src/components/RichTextEditor/Plugins/ColumnBreakPlugin/utils/setColumnBreaks.ts

```diff
@@ -1,29 +1,24 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import {
-    ENode,
-    PlateEditor,
-    PlatePluginKey,
-    SetNodesOptions,
-    TNodeMatch,
-    Value,
-    isBlock,
-    setElements,
-    unsetNodes,
-} from '@udecode/plate';
-import { Location } from 'slate';
+import { ENode, SetNodesOptions, TLocation, TNodeMatch, Value, isBlock, setElements, unsetNodes } from '@udecode/slate';
+import { PlateEditor, PlatePluginKey } from '@udecode/plate-core';
 import { KEY_ELEMENT_BREAK_AFTER_COLUMN } from '../createColumnBreakPlugin';
 
 export type ColumnBreakValue = 'active' | 'inactive' | undefined;
 
+type SetColumnBreaksType = <V extends Value>(
+    editor: PlateEditor<V>,
+    { key, value, at }: { at?: TLocation; value?: string; setNodesOptions?: SetNodesOptions<V> } & PlatePluginKey,
+) => void;
+
```

Comment:
AFAIK this isn't required here, no need to duplicate the typing of the function since the props are already typed